feat: add 1.0 blog post

toddbaert opened the pull request:
Add blog post about 1.0 and spec stabilization.

Direct preview link: https://deploy-preview-160--lucky-creponne-baf9b3.netlify.app/blog

tcarrio commented on blog/21-10-2022-hardening-and-1-0-sdks.md:
```diff
@@ -0,0 +1,35 @@
+
+Early this year, OpenFeature announced its intent to bring a standard to the rapidly growing development practice of feature flagging.
+In June it was accepted as a Cloud Native Computing Foundation Sandbox project.
+Now, we're pleased to announce a new milestone: OpenFeature has released 1.0 versions of it's [.NET](https://github.com/open-feature/dotnet-sdk), [Go](https://github.com/open-feature/go-sdk), [Java](https://github.com/open-feature/java-sdk), and [JavaScript](https://github.com/open-feature/js-sdk) SDKs!
```
s/it's/its/

toddbaert replied:
Nice catch, thanks!

tcarrio commented on blog/21-10-2022-hardening-and-1-0-sdks.md:
```diff
@@ -0,0 +1,35 @@
+## What's next?
+
+Our goal in the upcoming months will be to harden our existing experimental features. Additionally, we'll work to develop and standardize new capabilities, including: client-side feature flagging, improved cloud native tooling, and implicit transaction-scoped data propagation of contextual attributes.
+Furthermore, we're working on SDKs for additional languages, including PHP, Python, and Ruby.
```
Perhaps link to these SDK repositories

toddbaert replied:
Looks like you did this.

tcarrio reviewed (approved):
Grammar and possible linking opportunities

tcarrio commented:
Additionally, this is a preemptive post, correct? In anticipation of the 1.0 spec release? Asking since I still don't see a 1.0 in the spec repo :slightly_smiling_face:

If `0.5.0` is essentially `1.0` that would be good to know as well

tcarrio commented on blog/21-10-2022-hardening-and-1-0-sdks.md:
```diff
@@ -0,0 +1,35 @@
+## What's next?
+
+Our goal in the upcoming months will be to harden our existing experimental features. Additionally, we'll work to develop and standardize new capabilities, including: client-side feature flagging, improved cloud native tooling, and implicit transaction-scoped data propagation of contextual attributes.
+Furthermore, we're working on SDKs for additional languages, including PHP, Python, and Ruby.
```
```suggestion
Furthermore, we're working on SDKs for additional languages, including [PHP](https://github.com/open-feature/php-sdk), [Python](https://github.com/open-feature/python-sdk), and [Ruby](https://github.com/open-feature/ruby-sdk).
```

toddbaert commented:
> Additionally, this is a preemptive PR, correct? In anticipation of the 1.0 spec release? Asking since I still don't see a 1.0 in the spec repo slightly_smiling_face
> 
> If `0.5.0` is essentially `1.0` that would be good to know as well

We are not doing a 1.0 release of the spec, but have added `hardening` labels to certain subsections. Only the mentioned SDKs are 1.0 (2/4 thus far, but hopefully it's 4/4 soon). Should we add clarification on that?

justinabrahms commented:
I want to find a 3rd bullet point for the experimental sdk features. Also, "hardening" sounds almost stable but not actually stable. Can it regress? If not, we should find a better name. If so, we should mention it. :)

toddbaert commented:
> I want to find a 3rd bullet point for the experimental sdk features.

I'm sure I can think of one.

> Also, "hardening" sounds almost stable but not actually stable. Can it regress? If not, we should find a better name. If so, we should mention it. :)

The "hardening" language was agreed on here: https://github.com/open-feature/spec/pull/139. It can't be regressed; we agreed we wanted a unidirectional graduation.

I'm open to changing the term if the meaning doesn't change. Any proposals?

justinabrahms commented:
This works for me. Re read that post and it's about which version number changes on update, which works for me.

toddbaert commented:
@justinabrahms 

I've added one more point to the experimental features. Right now, we have 4 sections in the SDK, 2 of which are `hardening` and 2 of which are sill `experimental`. The point I've added is "experimental context propagation" (passing context attrs on the async continuation or thread-local storage, etc) which is currently only implemented experimentally in the JS-SDK.
